driver-core/src/test/unit/com/mongodb/client/model/bulk/BaseClientDeleteOptionsTest.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.client.model.bulk;
+
+import com.mongodb.MongoBaseInterfaceAssertions;
+import org.junit.jupiter.api.Test;
+
+class BaseClientDeleteOptionsTest {
+
+    @Test
+    void testAllSubInterfacesOverrideMethods() {
+        MongoBaseInterfaceAssertions.assertSubtypeReturn("com.mongodb", BaseClientDeleteOptions.class);
```

Comment:
Do we really need to pass `"com.mongodb"` here? Maybe we could hardcode it in `assertSubtypeReturn`, as we will never pass anything else.